services/storage/src/simcore_service_storage/datcore_wrapper.py

```diff
@@ -64,19 +85,36 @@ def delete_file(self, destination: str, filename: str):
         # the object can be found in dataset/filename <-> bucket_name/object_name
         try:
             self.d_client.delete_file(destination, filename)
-        except Exception as e:
-            logger.exception("Error deleting datcore file %s", e)
+        except Exception:
+            logger.exception("Error deleting datcore file")
+
+    @make_async
+    def delete_file_by_id(self, file_id: str):
+        try:
+            self.d_client.delete_file_by_id(file_id)
+        except Exception:
+            logger.exception("Error deleting datcore file")
 
     @make_async
     def download_link(self, destination: str, filename: str):
         url = ""
         try:
             url = self.d_client.download_link(destination, filename)
-        except Exception as e:
-            logger.exception("Error getting datcore download link %s", e)
+        except Exception:
+            logger.exception("Error getting datcore download link")
 
         return url
 
+    @make_async
+    def download_link_by_id(self, file_id: str):
```

Comment:
The implementation of these functions is quite consistent, I guess whether we could automate them by either using a template or some runtime reflection